src/components/EventCard.tsx

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+import { ReactElement } from "react";
+import ".././styles.css";
+import PinSVG from ".././public/Pin.svg";
+import TextIconSVG from ".././public/TextIcon.svg";
+import MeatballMenuSVG from ".././public/MeatballMenu.svg";
+
+export interface EventCardProps {
+  startTime: Date;
+  endTime?: Date;
+  name: string;
+  location: string;
+  description: string;
+}
+
+/**
+ * Renders a single event in the feed
+ */
+const EventCard = ({
+  startTime,
+  endTime,
+  name,
+  location,
+  description,
+}: EventCardProps): ReactElement => {
+  let startTimeString: string = startTime.toLocaleString("en-US", {
+    hour: "numeric",
+    minute: "numeric",
+    hour12: true,
+  });
+  let endTimeString: string = endTime
+    ? endTime.toLocaleString("en-US", {
+        hour: "numeric",
+        minute: "numeric",
+        hour12: true,
+      })
+    : "";
+
+  return (
+    <>
+      <div className="pl-4 pt-4 pr-6">
+        <span className="font-sans">
+          {startTimeString + (endTime ? " to " + endTimeString : "")}
+        </span>
+        <div className="flex flex-row justify-between items-start mb-4">
+          <div className="not-italic font-bold text-2xl leading-8 font-sans">
+            {name}
+          </div>
+          <img src={MeatballMenuSVG} alt="Menu svg" />
+        </div>
+        <div className="flex flex-row items-start">
+          <img src={PinSVG} alt="Pin svg" className="p-1 pt-0" />
+          <span className="text-body-mobile pl-2 pr-4 mb-4 font-montserrat">
```

Comment:
Here I'd add the margins to the parent flexbox rather than this element. It makes it easier when resizing the flexboxes for different screen sizes, since you may end up needing to manually finetune this bottom margin space on larger screens. Instead you can add a margin to the parent flexbox and if you want to keep the text lower than the icon, use `align-self: center` on the icon. (I'd do the same for the description element too!)